### PR TITLE
Evaluation of php-substrate-api M2

### PR DESCRIPTION
**Pull request by keeganquigley**
Delivery: https://github.com/w3f/Grant-Milestone-Delivery/pull/434

Application: https://github.com/w3f/Grants-Program/pull/622

### CODE REVIEW

**Comment by takahser** [comment]
@keeganquigley thanks a lot for conducting this evaluation!
Just 1 question, I saw that 1 test currently fails. Ideally, the grantee should fix this. Is there any reason not to fix it or to assume that it's not an actual failure?

File: evaluations/php-rpc-lib_2_keeganquigley.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Evaluation
+
+- **Status:** Accepted
+- **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/php-rpc-lib.md
+- **Milestone:** 2
+- **Previously successfully merged evaluation:** Alxs, Keeganquigley
```
Comment (takahser) [inline evaluations/php-rpc-lib_2_keeganquigley.md:6]:
```suggestion
- **Previously successfully merged evaluation:** All by Keeganquigley
```

Comment (takahser) [reply evaluations/php-rpc-lib_2_keeganquigley.md:6]: FYI - here we refer to evaluations that the author (in this case you) has already conducted and was merged.

**Comment by keeganquigley** [comment]
Thanks @takahser, it was an issue with the FFI libraries. Grantee has addressed both errors and updated troubleshooting info in the README. The build now deploys successfully, locally and via the [GitHub CI workflow](https://github.com/gmajor-encrypt/php-substrate-api/actions/runs/2301584908). I believe we are good to go now. I have updated the eval.

**Review by takahser — APPROVED** [review]
@keeganquigley thanks a lot!

**Pull request merged**